packages/core/src/Stores/client-store.js

```diff
@@ -2933,7 +2933,9 @@ export default class ClientStore extends BaseStore {
     }
 
     get is_account_to_be_closed_by_residence() {
-        return this.account_time_of_closure && this.residence && this.residence === 'sn';
+        return this.account_status?.account_closure?.find(
+            item => item?.status_codes?.includes('residence_closure') && item?.type === 'residence'
```

Comment:
Isn't it exactly the function above `account_time_of_closure` ?

Reply:
account_time_of_closure has this check in the end, ?.time_of_closure, which we don't need in our case. We just need to check if .includes('residence_closure') && item?.type === 'residence'